src/Features/CSharp/Portable/Completion/CompletionProviders/ObjectCreationCompletionProvider.cs

```diff
@@ -79,6 +79,17 @@ protected override (string displayText, string insertionText) GetDisplayAndInser
                 return (symbol.Name, symbol.Name);
             }
 
+            // If not all  of the type arguments are concrete types, eg
+            // we've inferred something like `Task<T>`, just suggest the
+            // generic type `Task<>`
```

Comment:
this comment doesn't give te "why".  "why" would we want to do this?

It seems completely reasonable to both infer and suggest "Task<T>" if that's appropriate to the context we're in.